### PR TITLE
Feature/implement read url

### PR DESCRIPTION
**Pull request by Amet015**

### CODE REVIEW

File: core/api/request_api/RequestApi.py [hunk 1]
```diff
@@ -24,13 +24,16 @@ def verify_value(self, row_key, row_value, response_param):
...
+        if data_table is not None:
+            for row in data_table:
+                row_key = row['key']
+                row_value = row['value']
+                row_value = self.verify_value(row_key, row_value, response_param)
+                data_dictionary[row_key] = row_value
+
         data_dictionary["key"] = self.config.get_config_file()['key']
         data_dictionary["token"] = self.config.get_config_file()['token']
+
```
Comment (JuanCarlosMartinez4) [inline core/api/request_api/RequestApi.py]: delete this line of code

File: core/api/request_api/RequestApi.py [hunk 2]
```diff
@@ -39,18 +42,15 @@ def do_request(self, http_type, input_endpoint, data_table, response_param):
...
-        if http_type == "POST":
-            self.response = requests.request(http_type, url, data=body_content, headers=HEADERS)
-        else:
-            values["key"] = self.config.get_config_file()['key']
-            values["token"] = self.config.get_config_file()['token']
-            url += self.verify_value_delete(response_param)
-            self.response = requests.request(http_type, url, params=values)
+
+        self.response = requests.request(http_type, url, data=body_content, headers=HEADERS)
+
```
Comment (JuanCarlosMartinez4) [inline core/api/request_api/RequestApi.py]: delete this line of code

File: core/api/request_api/RequestApi.py [hunk 3]
```diff
@@ -39,18 +42,15 @@ def do_request(self, http_type, input_endpoint, data_table, response_param):
...
         return self.response
 
-    def verify_value_delete(self, response_param):
-        key = 'BoardObject'
-        row_value = response_param.get(key, None)
-        return row_value
+    def replace_variables(self, input_endpoint, response_param):
+        for key in response_param:
+            if input_endpoint.find(key) > -1:
```
Comment (JuanCarlosMartinez4) [inline core/api/request_api/RequestApi.py]: try to avoid to use magic numbers

File: core/api/request_api/RequestApi.py [hunk 4]
```diff
@@ -24,33 +24,30 @@ def verify_value(self, row_key, row_value, response_param):
...
+
+        self.response = requests.request(http_type, url, data=body_content, headers=HEADERS)
         return self.response
 
-    def verify_value_delete(self, response_param):
-        key = 'BoardObject'
-        row_value = response_param.get(key, None)
-        return row_value
+    def replace_variables(self, input_endpoint, response_param):
+        NO_FOUND = -1
```
Comment (JuanCarlosMartinez4) [inline core/api/request_api/RequestApi.py]: this variable is NOT_FOUND

File: core/api/request_api/RequestApi.py [hunk 5]
```diff
@@ -24,33 +24,30 @@ def verify_value(self, row_key, row_value, response_param):
 
     def generate_data(self, data_table, response_param):
```
Comment (Enrique-C) [inline core/api/request_api/RequestApi.py]: response_param would be id_dictionary

**Review by Enrique-C — APPROVED** [review]

**Review by JuanCarlosMartinez4 — APPROVED** [review]

**Pull request merged**